src/internal/operators/mergeMapTo.ts

```diff
@@ -3,7 +3,7 @@ import { mergeMap } from './mergeMap';
 import { ObservableInput } from '../types';
 
 /* tslint:disable:max-line-length */
-export function mergeMapTo<O extends ObservableInput<any>>(innerObservable: O, concurrent?: number): OperatorFunction<any, ObservedValueOf<O>>;
+export function mergeMapTo<T, O extends ObservableInput<any>>(innerObservable: O, concurrent?: number): OperatorFunction<any, ObservedValueOf<O>>;
```

Comment:
```suggestion
export function mergeMapTo<T, O extends ObservableInput<any>>(innerObservable: O, concurrent?: number): OperatorFunction<T, ObservedValueOf<O>>;
```

Should this not be `T` here also?

Reply:
I thought I'd reverted the signature changes and included only the tests that were added. These changes to the signatures should not be there. The signatures they are replacing are correct. Lemme see if I can clean this PR up a bit.